### DISCUSSION TITLE
Google Analytics - Transition from UA to GA4

### DISCUSSION THREAD

**Discussion by thehoffm** [YOURLS/YOURLS#3343] [Plugins]
Looking at the current plugins, there's only one I can see that tracks links in Google Analytics:
https://github.com/powerthazan/YOURLS-GA-MP-Tracking

The issue with that plugin is that it's written for Google's Universal Analytics protocol which is transitioning to Google Analytics 4 on July 1st, meaning the plugin will stop working then.

I'd like to see a plugin that will work for GA4 with Yourls on-going.

**Comment by ozh** [comment]
I think you may want to open an issue in the plugin' repo

**Reply by thehoffm** [reply]
Done. I also reached out to the dev on twitter.

**Reply by JemJal** [reply]
Did anyone reply? I am really interested in this as well.
